### PR TITLE
Fix exception

### PR DESCRIPTION
**Pull request by sbellem**
The original goal of this PR was to resolve issue #35.

In addition to solving the above, tests were added for the `bigchaindb` command.

### CODE REVIEW

File: tests/test_commands.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    monkeypatch.setattr(db, 'init', mockreturn)
+
+
+@pytest.fixture
+def mock_processes_start(monkeypatch):
+    from bigchaindb.processes import Processes
+    monkeypatch.setattr(Processes, 'start', lambda *args: None)
+
+
+@ pytest.fixture
```
Comment (vrde) [inline tests/test_commands.py]: Remove whitespace between `@` and `pytest.fixture`

**Comment by vrde** [comment]
:+1:

File: tests/test_commands.py (hunk 1 above)
Comment (sbellem) [inline tests/test_commands.py]: Done, thanks!

**Comment by vrde** [comment]
:+1:, and thanks for improving the coverage.

**Comment by sbellem** [comment]
@vrde Cool!

**Pull request merged**